src/Uplink/API/V3/Auth/Token_Authorizer_Cache_Decorator.php

```diff
@@ -71,7 +72,7 @@ public function is_authorized( string $license, string $slug, string $token, str
 			return false;
 		}
 	
-		set_transient( $transient, $is_authorized ? 1 : 0, $this->expiration );
+		set_transient( $transient, $is_authorized ? 'authorized' : 'not_authorized', $this->expiration );
```

Comment:
In this case I'd make this public constants for the class and uses them directly in the tests